src/Blocks/Popover.purs

```diff
@@ -0,0 +1,46 @@
+module Ocelot.Block.Popover where
+
+import Prelude
+
+import Ocelot.Block.Hover as Hover
+import DOM.HTML.Indexed (HTMLdiv)
+import Halogen.HTML as HH
+import Halogen.HTML.Properties as HP
+
+popover
+  :: forall p i
+   . Array (HH.IProp HTMLdiv i)
+  -> Array Hover.HoverAnchor
+  -> HH.HTML p i
+  -> Array (HH.HTML p i)
+  -> HH.HTML p i
+popover props anchors trigger hoverHtml =
+  Hover.hover
+  props
+  anchors
+  trigger
+  ( HH.div
+    [ HP.classes hoverClasses ]
+    hoverHtml
+  )
```

Comment:
Can you indent from line 19 to make it clearer that these are all params to `Hover.hover`?

Reply:
I don't know how I overlooked that >_<. It's fixed, thanks!